src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -597,7 +597,24 @@ export class Connector implements IConnector {
             }
```

Comment:
@qzhou1607 Will change the commit message to use the [`Chore:`](https://sonarwhal.com/docs/developer-guide/contributing/pull-requests.html#commitmessages) tag as this is not a user-facing thing).